package.json

```diff
@@ -326,7 +326,7 @@
     "@metamask/snaps-sdk": "^6.19.0",
     "@metamask/snaps-utils": "^9.0.1",
     "@metamask/solana-wallet-snap": "^1.11.0",
-    "@metamask/transaction-controller": "^48.1.0",
+    "@metamask/transaction-controller": "file:.yalc/@metamask/transaction-controller",
```

Comment:
This dependency will need update I think.

Reply:
Mentioned in the description that both releases are in progress 👍

Reply:
Updated.